Assets/Scripts/Drinkables/AbstractDrinkable.cs

```diff
@@ -19,6 +19,7 @@ public abstract class AbstractDrinkable : MonoBehaviour
         protected void CommonDrunkennessEffects()
         {
             // All beers will have a base score multiplier of 2
+            // TODO: Update to base score multiplier directly off intoxication level instead.
```

Comment:
Why would you move this to the intoxication level? Doesn't the multiplier only change when the player drinks?